docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -36,24 +36,17 @@ plugins {
-|----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| _&lt;targetName&gt;_ | Declares a particular target of a project. The names of available targets are listed in the [Targets](#targets) section.                                                                                                        |
-| `targets`            | All targets of the project.                                                                                                                                                                                                     |
-| `presets`            | All predefined targets. Use this for [configuring multiple predefined targets](#targets) at once.                                                                                                                               |
-| `sourceSets`         | Configures predefined and declares custom [source sets](#source-sets) of the project.                                                                                                                                           |
-| `compilerOptions`    | Extension-level common [compiler options](gradle-compiler-options.md) that are used as defaults for all targets and shared source sets. To use it, add the following opt-in: `@OptIn(ExperimentalKotlinGradlePluginApi::class)` |
-
-> The support for `compilerOptions {}` as a top-level block is [Experimental](components-stability.md#stability-levels-explained)
-> and requires opt-in. It may be dropped or changed at any time. Use it only for evaluation purposes. We would appreciate
-> your feedback on it in [YouTrack](https://kotl.in/issue).
->
-{style="warning"}
+| **Block**            | **Description**                                                                                                                |
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| _&lt;targetName&gt;_ | Declares a particular target of a project. The names of available targets are listed in the [Targets](#targets) section.       |
+| `targets`            | All targets of the project.                                                                                                    |
+| `sourceSets`         | Configures predefined and declares custom [source sets](#source-sets) of the project.                                          |
+| `compilerOptions`    | Extension-level common [compiler options](#compiler-options) that are used as defaults for all targets and shared source sets. |
```

Comment:
```suggestion
| `compilerOptions`    | Specifies common extension-level [compiler options](#compiler-options) that are used as defaults for all targets and shared source sets. |
```

The word sequence "common extension-level options" sounds better to me than "extension-level common options" for some reason, but feel free to ignore that :)